pkgs/development/tools/analysis/include-what-you-use/default.nix

```
@@ -10,7 +10,11 @@ stdenv.mkDerivation rec {
     url = "${meta.homepage}/downloads/${pname}-${version}.src.tar.gz";
   };
 
-  nativeBuildInputs = with llvmPackages; [ cmake llvm.dev llvm python3];
+  postPatch = ''
+    patchShebangs .
+  '';
+
+  nativeBuildInputs = with llvmPackages; [ cmake llvm.dev llvm python3 ];
   buildInputs = with llvmPackages; [ libclang clang-unwrapped ];
```

```suggestion
  buildInputs = with llvmPackages; [ libclang clang-unwrapped python3 ];
```

so when cross-compiling the files will have host shebangs